2-mandatory/2-financial-times.js

```diff
@@ -5,33 +5,68 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    headlines = []; // we declare an empty array so when we get the headlines with length<=65 we push them here
+
+    for (article of allArticleTitles) { // we loop through all articles  and check their length
```

Comment:
Small suggestion: it is usually a good idea to declare the variable in the `for` loop with let or const:
`for (const article of allArticleTitles) {`